src/Settings/Plugins.php

```diff
@@ -9,42 +9,50 @@ public function render_Plugins_settings() {
             array(
                 'label' => 'Wordpress',
                 'logo' => 'wordpress_logo.png',
-                'options' => array('Login', 'Register', 'Comments', 'Forgot Password')
+                'options' => array('Login', 'Register', 'Comments', 'Forgot Password'),
+                'message' => ''
             ),
             array(
                 'label' => 'Woocommerce',
                 'logo' => 'woocommerce_logo.png',
-                'options' => array('Login', 'Register', 'Forgot Password')
+                'options' => array('Login', 'Register', 'Forgot Password'),
+                'message' => ''
             ),
             array(
                 'label' => 'ContactForm7',
                 'logo' => 'contactForm7_logo.png',
-                'options' => array('Forms')
+                'options' => array('Forms'),
+                'message' => array('For manual integration, see the', 'documentation.', 'https://docs.adcaptcha.com/wordpress/mailchimp')
```

Comment:
Can you change the url to me https://docs.adcaptcha.com/wordpress/contactform7

I dont know if this should be the message, what do you think @MattGibney ?

"Some extra steps are requite, see the docs."

Reply:
Link is updated.

Reply:
<img width="1099" alt="Screenshot 2024-11-19 at 11 25 09" src="https://github.com/user-attachments/assets/5eaf5057-dd2d-4f51-a521-e11d9b8d1198">

Reply:
<img width="1179" alt="Screenshot 2024-11-19 at 12 18 46" src="https://github.com/user-attachments/assets/24d61519-8eaf-41f0-b139-12f49bef782c">